app/utils/localAuthentication.js

```diff
@@ -103,7 +104,11 @@ export const localAuthenticate = async(server) => {
 	// if screen lock is enabled
 	if (serverRecord?.autoLock) {
 		// Make sure splash screen has been hidden
-		await RNBootSplash.hide();
+		try {
+			await RNBootSplash.hide();
+		} catch (error) {
+			logEvent(events.LC_SERVER_RECORD_BOOT_SPLASH_HIDE_ERROR);
```

Comment:
@reinaldonetof This is wrong imo.
If we want to log something, let's log the error.